Hunting Queries/OracleDatabaseAudit/OracleDBAuditListOfTablesQueried.yaml

```diff
@@ -0,0 +1,26 @@
+id: b8fcb5b8-254c-4d49-865e-403b8453f487
+name: OracleDBAudit - Top tables queries
+  'Query searches for tables queries.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - Collection
+  - Impact
+query: |
+  let lbtime = 24h;
+  OracleDatabaseAuditEvent
+  | where TimeGenerated > ago(lbtime)
+  | where DbAction =~ 'SELECT'
+  | extend TableName = replace(@'[,\(\)]', '', extract(@'(?i)SELECT(.*?)FROM\s(.*?)\s', 2, Action))
+  | where isnotempty(TableName)
+  | where TableName !in ('SELECT', 'select')

```

Comment:
use `| where TableName !~ 'select'` <-- case insensitive match

Reply:
updated

Reply:
looks like it did not get updated

Reply:
fixed